web/tools/webpack.config.babel.js

```diff
@@ -111,7 +111,7 @@ const createConfig = (env = {}) => {
     // What information should be printed to the console
     stats: 'minimal',
     performance: {
-      hints: isProductionBuild ? 'error' : false,
+      hints: isProductionBuild && !devServer ? 'error' : false,
```

Comment:
Isn't !devServer always true when isProducitonBuild is true?

Reply:
Not for malte, we only have one build config here. Maybe we should add one though at some point.

Reply:
But why do we not want to show hints when we are running as dev server?

Reply:
Oh because it is bigger, right?

Reply:
Yes, otherwise compiling fails.

Reply:
Hm maybe we should add this configh in this pr. I think it is not good that we can run a dev-server and buildConfig().development is false. Or should we do this in the Bug you opened this week about the bundle size?

Reply:
Separate PR please because other PRs depending on this :) Will do in the opened bug